frontend/src/libs/utils.js

```diff
@@ -11,3 +11,14 @@ export function strip(x) {
     }
     return ("" + x).trim();
 }
+
+/**
+ * Formats the input date string to be human readable
+ * Input string is of the form 2019-01-01T00:00:00.000Z
+ * Output string is of the form January 1, 2019
+ * @param {string} dateString 
+ */
+export function formatDate(dateString) {
+    const date = new Date(dateString);
+    return `${date.toLocaleDateString("en-US", { month: 'long', year: 'numeric', day: 'numeric' })}`;
```

Comment:
Let's use the Canadian locale. Also, 3-letter months is more compact.

Reply:
I've made this change